docs/Secure-Coding-Guide-for-Python/readme.md

```diff
@@ -1,36 +1,37 @@
 # Secure Coding One Stop Shop for Python
 
-Promote secure products by knowing the difference between secure compliant
-and non-compliant code with `CPython >= 3.9` using modules listed on
+An initiative by the OpenSSF to provide new Python programmers a resource to study secure coding in `CPython >= 3.9` with working code examples.
 
-[Python Module Index](https://docs.python.org/3.9/py-modindex.html) [Python 2023].
+Documentation is written in academic style to support security researchers while using in plain English to cater for an international audience.
 
-This page is an initiative by the OpenSSF to improve secure coding in Python by providing a location for study. Its structure is based on
-Common Weakness Enamurator (CWE) [Pillar Weakness](https://cwe.mitre.org/documents/glossary/#Pillar%20Weakness) [mitre.org 2023].
-
-Some rules only contain code examples, documentation will follow.
+Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)], languages such as `HTML` and `JavaScript` are avoided.
```

Comment:
```suggestion
Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)] are not specifically covered by this document. We do sometimes note where outside modules are necessary to address a type of vulnerability, and encourage users of those modules to consult those modules' documentation.
```

Reply:
The note about HTML & JavaScript isn't really the issue. The point is that this document only discusses the built-in modules. Presumably JSON *would* be in scope, because there's a built-in module for JSON.

Reply:
@david-a-wheeler , the background story to this learning resource is that we have designers spending 40 hours going through a ton of text to learn secure coding in languages such as JAVA. Any chance we can shorten that sentence?

Reply:
```suggestion
Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)] are not specifically covered by this document.
```

Reply:
How about this? The issue isn't really HTML (there's an `html` module in Python's built-ins) - it's that this particular document only focuses on the built-ins.

Reply:
Much better thank you.

Since I am not native English speaking I struggle to understand the difference between "avoiding" and "not specifically covered". We try our best to only list "inclusions" in order to have ever expanding lists of "exclusions". Not sure how to best phrase that in this introduction.

We are missing  a contribution document,  I will get onto creating that. In it I would outline the following in regards to explaining why we avoid modules outside of the standard lib:
* Code examples may end up as module recommendations "just use..."
* Actual issue is not explained in code.
* Don't want to end up writing module documentation, bug-fixes or workarounds.
* Life Cycle related issues
* Trust related issues

We also want to keep the main pySCG readme.md look "inviting" with a minimum of content and links in its introduction section.